shared/config/socket.ts

```diff
@@ -1,7 +1,8 @@
 import { ServerOptions } from 'socket.io';
+import { DEFAULT_API_SOCKET_PORT, DEFAULT_CORE_SOCKET_PORT, DEFAULT_CORE_RPC_PORT } from 'shared/config';
 
 export const API_HOST = process.env.SERVER_HOST || 'localhost';
```

Comment:
`localhost` is DEFAULT_PROPERTY?

Reply:
This constant never used, I think it must be deleted. And I will add `SERVER_HOST` as a required environment variable in z-schema validation.